The white space between ']' and ']' is necessary or else flex will discard
all closing braces and brackets of the corresponding line in docify.cc.
Patch required with flex version 2.5.31, not required with 2.5.4a.

--- doc++-3.4.10/src/docify.ll	2001-03-08 08:36:20.000000000 +0000
+++ doc++-3.4.10-patched/src/docify.ll	2004-01-13 22:12:27.000000000 +0000
@@ -314,7 +314,7 @@
 	{
 	if(argv[1][0] == '-')
 	    {
-	    fprintf(stderr, "Usage: docify [infile [outfile]]\n");
+	    fprintf(stderr, "Usage: docify [ infile [ outfile ] ]\n");
 	    return 0;
 	    }
 	ain = fopen(argv[1], "r");
